base/libgit2/libgit2.jl

```diff
@@ -95,6 +95,17 @@ Checks if there have been any changes to tracked files in the working tree (if
 `cached=false`) or the index (if `cached=true`).
 `pathspecs` are the specifications for options for the diff.
 
+# Example
+```julia
+repo = LibGit2.GitRepo(repo_path)
+LibGit2.isdirty(repo) #should be false
```

Comment:
(I'd just edit, but you might disagree)
I think comments are a little more readable with a space after the `#`